src/main/java/com/labsynch/labseer/utils/MimeTypeUtil.java

```diff
@@ -26,7 +26,7 @@ public static String getContentTypeFromExtension(String originalFilename) {
 			mimeTypeStream = ApiFileSaveController.class.getClassLoader().getResourceAsStream(MIME_PROP_FILE);
 			Properties mimeTypeProps = new Properties();
 			mimeTypeProps.load(mimeTypeStream);
-			contentType = mimeTypeProps.getProperty(fileExtension);
+			contentType = mimeTypeProps.getProperty(fileExtension.toLowerCase());
```

Comment:
> Does "abc.PDF" overwrite "abc.pdf"?
No, these files are considered distinct and would not overwrite.

>In case the system doesn't do that, then this is going to be tricky because I think we'll still return "abc.pdf" when the user next queries "abc.pdf" or "abc.PDF"?
This function is not actually used for reading the file. This function is just is just for getting the mime type of the file so it should have any affect on the actual file content return.  https://github.com/mcneilco/acas-roo-server/blob/a95be8232b476d228d412c0fde4c9059e9d03f3c/src/main/resources/mimeType.properties#L28

Reply:
I see, thanks for confirming 👍